content/guides/references/migration-guide.md

```diff
@@ -83,6 +83,170 @@ on('<event>', (arg1, arg2) => {
 
 CONTENT_TBD
 
+## Migrating to Cypress 9.3
```

Comment:
Instead of having this content in the Migration Guide, which is really geared toward mandatory changes, can you add a section called "Upgrading from the cypress-file-upload plugin" to the `.selectFile()` API docs?

Reply:
I prefer the solution we used with https://docs.cypress.io/guides/references/migration-guide#Migrating-cy-route-to-cy-intercept, where it's in the migration guide but not attached to a specific release. Moving it into the selectFile docs would clutter up our API docs with a guide that's not of interest to new users or old users once they've gone through it once. It also feels like it would be harder to find in there.